src/lighteval/models/model_config.py

```diff
@@ -207,13 +209,14 @@ def init_configs(self, env_config: EnvConfig):
 @dataclass
 class VLLMModelConfig:
     pretrained: str
```

Comment:
This class could use some doc on how best to choose the params

Reply:
For parallelism, I think users go for TP, then PP for big models, and DP for small models, and we could add some info about this somewhere (or any other heuristic)